test/Microsoft.AspNetCore.Razor.Test.MvcShim/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewDataDictionary.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures
+{
+    public class ViewDataDictionary : IDictionary<string, object>
+    {
+        public object Model { get; set; }
```

Comment:
I don't beleive the codegen uses any of the properties on ViewDataDictionary. I'd comb over it to see what can be nuked. Also, do we ever use the non generic variant?